src/components/FlyoutMenu/FlyoutMenu.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+
+import FlyoutMenu from './FlyoutMenu';
+
+describe('FlyoutMenu', () => {
```

Comment:
I would also test the callbacks, and you can also target the testIDs instead of the classes.

Reply:
Yes - also using `testing-library/react` and `testing-library/dom` instead of `enzyme` would be preferred.

Reply:
@tay1orjones Oh, I didn't know that. How come, are we moving away from enzyme?